cerbero/build/filesprovider.py

```diff
@@ -123,8 +123,8 @@ class FilesProvider(object):
     # UNIX shared libraries can have between 0 and 3 version components:
     # major, minor, micro. We don't use {m,n} here because we want to capture
     # all the matches.
-    _SO_REGEX = r'^lib{}\.so(\.[0-9]+)?(\.[0-9]+)?(\.[0-9]+)?$'
-    _DYLIB_REGEX = r'^lib{}(\.[0-9]+)?(\.[0-9]+)?(\.[0-9]+)?\.dylib$'
+    _SO_REGEX = r'^{}\.so((\.[0-9]+)+)?$'
```

Comment:
Is this change because of dll's not starting with lib? There is a specific regex expresion for DLL's IIRC

Reply:
This comes from commit 237c5404a13628871e85effac027bcb101fd286b:
>There are some library names that do not start with "lib", such as preloadable_libintl.so

Filesprovider is irrelevant now for fridge since we don't use it as we package every single file that is installed, but the regex shouldn't be as strict because there are quite a few exceptions.